h2o-py/tests/testdir_algos/anovaglm/pyunit_PUBDEV_8088_NA_handling.py

```diff
@@ -0,0 +1,29 @@
+import sys
+sys.path.insert(1,"../../../")
+import h2o
+from tests import pyunit_utils
+from h2o.estimators.anovaglm import H2OAnovaGLMEstimator
+
+# Simple test to check correct NA handling skip.
+def testFrameTransform():
+  train = h2o.import_file(path=pyunit_utils.locate("smalldata/prostate/prostate_complete.csv.zip"))
+  myY = 'CAPSULE'
+  myX = ['AGE','VOL','DCAPS']
+  train[10,2] = None
+  train[20,7] = None
+  # build model choosing skip
+  anovaG1 = H2OAnovaGLMEstimator(family='binomial', Lambda=0, missing_values_handling="skip")
```

Comment:
Lambda parameter is deprecated

Reply:
Ah, good point.  I changed it to lambda_.  Thanks.